sos/policies/distros/redhat.py

```diff
@@ -206,6 +206,32 @@ def get_tmp_dir(self, opt_tmp_dir):
 
 
 class RHELPolicy(RedHatPolicy):
+    """
+    The RHEL policy is used specifically for Red Hat Enterprise Linux, of
+    any release, and not forks or derivative distributions. For example, this
+    policy will be loaded for any RHEL 8 installation, but will not be loaded
+    for CentOS Stream 8 or Red Hat CoreOS, for which there are separate
+    policies.
+
+    Plugins activated by installed packages will only be activated if those
+    packages are installed via RPM (dnf/yum inclusive). Packages installed by
+    other means are not considered by this policy.
+
+    By default, --upload will be directed to using the SFTP location provided
+    by Red Hat for technical support cases. Users who provide login credentials
+    for their Red Hat Customer Portal account will have their archives uploaded
+    to a user-specific directory.
```

Comment:
By default, `--upload` will rely on default `--upload-protocol=auto`, which will attempt an upload to the Red Hat Customer Portal (if credentials provided). Only when this fails or when credentials are missing, we failover to uploading to the SFTP server.

Reply:
I mean, starting the comment with "default is SFTP" sounds confusing to me, despite it is properly explained in the next paragraph.

Reply:
Yeah, this could go either way. The reason I went with saying SFTP is the 'default', is the is because we require 3 non-default options in order to attempt the customer portal.